apps/cowswap-frontend/src/common/pure/OrderSubmittedContent/index.tsx

```diff
@@ -38,6 +39,7 @@ export interface OrderSubmittedContentProps {
 }
 
 export function OrderSubmittedContent({ chainId, account, isSafeWallet, hash, onDismiss }: OrderSubmittedContentProps) {
+  const safeWallet = useGnosisSafeInfo()
```

Comment:
This is a pure component and shouldn't have hooks (other than local state).

Can you move this up to the closest non-pure parent?